client/webserver/site/src/js/markets.js

```diff
@@ -165,14 +165,21 @@ export default class MarketsPage extends BasePage {
     // Main order form
     forms.bind(page.orderForm, page.submitBttn, async () => { this.stepSubmit() })
     // Order verification form
-    forms.bind(page.verifyForm, page.vSubmit, async () => { this.submitOrder() })
+    forms.bind(page.verifyForm, page.vSubmit, async () => { this.submitOrder() }) // why not just ..., this.submitOrder)?
```

Comment:
Arrow functions preserve the scope of the declaration, so `this` will still mean `MarketsPage` in the function body and subsequently in `submitOrder`. If we didn't use an arrow function,`this` would depend on how the function was called. In the case of an event callback, I believe `this` is actually `undefined`.

Reply:
From my toying with `submitCancel`, it seems that `this` in an event handler is the event target (e.g. whatever was clicked).  Looking at the body of `submitOrder`, I can see why that's no good.  Before I saw the arrow function approach, I provided data to the handler via the event target object's data fields, which I'm actually happy with.